rig/routing_table/utils.py

```diff
@@ -0,0 +1,238 @@
+import warnings
+from rig.routing_table import RoutingTableEntry
+
+
+def table_is_subset_of(entries_a, entries_b):
+    """Check that every key matched by every entry in one table results in the
+    same route when checked against the other table.
```

Comment:
Is it worth pointing out that this function is likely only of use for testing? (i.e. regular programs shouldn't need it?) Is it worth considering putting it in a utils module in the test suite rather than the main API?